integration_tests/e2e/goal/review.cy.ts

```diff
@@ -94,6 +94,47 @@ context('Review goal(s)', () => {
     Page.verifyOnPage(AddNotePage)
   })
 
+  it('should be able to create another goal from the Review screen', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    const overviewPage = Page.verifyOnPage(OverviewPage)
```

Comment:
All this pre-amble in the // Given block is getting a bit long and repetitive - it's not your fault, and we don't have any better way of doing it at the moment, so don't worry; it's just be thinking out loud ....

I reckon what we need (NOT for this PR, we'll do it in a separate ticket) is a single cypress task or method called something like `prepareNewGoalAndArriveOnReviewGoalScreenFromOverviewScreen` (don't shoot me for the method name - I suspect we can do better 😁 ), and then we can simply have a test that does
```
It('should .....', () => {
  // Given
  cy.task('prepareNewGoalAndArriveOnReviewGoalScreenFromOverviewScreen')
  const reviewPage = Page.verifyOnPage(ReviewPage)

  // When
  ....
```

Anyway, don't worry about it for now - I'm just thinking out load ....

Reply:
Would `prepareReviewGoalScreen`, which creates one goal by default, suffice?

Reply:
Aye I was thinking it was getting quite long winded, I think the approach you suggested is a good one and as the service grows I imagine we'll need more of these resuable tasks!